components/browser/icons/src/main/java/mozilla/components/browser/icons/extension/IconMessageHandler.kt

```diff
@@ -28,7 +28,7 @@ internal class IconMessageHandler(
 ) : MessageHandler {
     private val scope = CoroutineScope(Dispatchers.IO)
 
-    @VisibleForTesting(otherwise = VisibleForTesting.NONE) // This only exists so that we can wait in tests.
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) // This only exists so that we can wait in tests.
```

Comment:
Was this needed for the gradle plugin? I'm fine with changing it since `PRIVATE` is accurate, but just curious.

(I don't see any errors locally)

Reply:
`lint` would fail, because `lastJob` is used in `loadRequest` method and the `NONE` visibility restricts calls from production code.